src/background/service/userWallet.ts

```diff
@@ -177,6 +177,39 @@ class UserWallet {
     this.loadAllAccounts(this.store.network, pubkey);
   };
 
+  /**
+   * Gets the keyindex of current account
+   * @returns Number the keyindex of current account
+   */
+  getKeyIndex = async () => {
+    try {
+      const matchingAccount = await this.getCurrentAccount();
+      const keyIndex = matchingAccount.keyIndex;
```

Comment:
Better add a handling, if there is no keyindex in matchingAccount. 
We can fallback to fetch it on keyindexer.